manager/integration/tests/test_basic.py

```diff
@@ -4558,3 +4561,66 @@ def test_backup_failed_disable_auto_cleanup(set_random_backupstore,  # NOQA
                       + volume_name + " is deleted"
     except AssertionError:
         pass
+
+
+def test_backup_save_access_mode(client, core_api, apps_api,  # NOQA
```

Comment:
Suggest renaming to `test_backup_volume_restore_with_access_mode`

Suggest having two test cases for rwx and rwo instead of mixing together. Probably you can leverage https://docs.pytest.org/en/7.1.x/example/parametrize.html to avoid duplicated code.

Reply:
I'm going to rename the function name and parameterize the parameters to have multiple scenarios.

https://github.com/longhorn/longhorn-tests/blob/c0f5ac9ca5589e641830fe791497c56494412de1/manager/integration/tests/test_basic.py#L4565-L4573